lambda-notes/src/components/Notes.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link, NavLink } from 'react-router-dom'
+import { Card, CardTitle, CardText, CardBody, Row, Col } from 'reactstrap';
+
+import { getNotes } from '../actions'
+import '../css/Notes.css';
+
+
+class Notes extends Component {
+    render() {
+        return (
+            <div className="notes">
+                    {!this.props.notes ?
+                        <h2> loading notes</h2>: null}
+                    <div style={{display:'flex', flexWrap:'wrap'}}>
+                    <Row >
+                        {this.props.notes.map((note) =>{
```

Comment:
Make sure you're giving your mapped components a `key` prop. Keys help React identify which components changed when it performs diffing of the original component tree with subsequent trees. You can read more about them here and why they're important: https://reactjs.org/docs/reconciliation.html#recursing-on-children